conda_forge_tick/migrators.xsh

```diff
@@ -89,6 +93,9 @@ class Migrator:
         """Branch to use on local and remote"""
         return 'bot-pr'
 
+    def migrator_hash(self, attrs):
+        return f'{self.__class__.__name__}_{self._class_version}'
```

Comment:
I'm trying to remember why we didn't go with a tuple for this. A tuple might make life easier.

Reply:
Or maybe a dict?

Reply:
You can always change it :)

Reply:
Ah, that is why, I can't put dicts into sets. I can put tuples though.

Reply:
But maybe I can live with lists since dicts seem nicer.

Reply:
A namedtuple might be what you want, actually

Reply:
Will `namedtuple` allow me to add keys as needed or do I need to put out a new one for each set of keys?